naming/README.md

```diff
@@ -0,0 +1,93 @@
+# Solana Naming Service [WIP]
+
+A spl program for issuing and managing ownership of: domain names, Solana Pubkeys, URLs, twitter handles, ipfs cid's etc..
+
+This program can be used for dns, pubkey etc lookups via a browser extension (wip), the goal is to create an easy way to identify Solana public keys with various links.
+
+Definitions: Names are strings that map to records which can hold multiple values.\
+ValueType defines the type (url, uri, solkey, etc..) of the value stored following the specification of the js bindings that will read from the name accounts.
```

Comment:
This would be a client-side spec. The on-chain program is basically use-case agnostic for better extensibility.

Reply:
Yes they could be anything and will only be interpreted by exterior applications. The specification for this could be defined in the js bindings to keep a greater flexibility after deployment. Let me know if I didn't understand your question!